File: Pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height, row, column, space;
    do
    {
        height = get_int("Height: ");
    }
    while ((height < 1) || (height > 8));

    // For each row
    for (row = 0; row < height; row++)
    {
        //For each column thats not part of the right aligned pyramide
        for (space = 0; space < height - row - 1; space++)
        {
            //Print a space
            printf(" ");
        }
        //For each column thats part of the right aligned pyramide
        for (column = 0; column <= row; column++)
        {
            //Print a brick
            printf("#");
        }
        //Create spacing between right and left aligned pyramides
        printf("  ");
        //For each column thats part of the left aligned pyramide
        for (column = 0; column <= row; column++)
        {
            //Print a brick
            printf("#");
        }
        //Move to next row
        printf("\n");
    }
}

File: Pset6/sentimental-mario-more/mario.py
# TODO

def main():

    height = get_height()

    for i in range(height):
        # Print right aligned pyramide
        for j in range(height):
            if j < height - i - 1:
                print(" ", end="")
            else:
                print("#", end="")
        # Print space between right and left aligned pyramide
        print("  ", end="")
        # Print left aligned pyramide
        for j in range(height):
            if j <= i:
                print("#", end="")
        # Move cursor to next line
        print()


def get_height():
    while True:
        try:
            n = int(input("Height? "))
            if ((n > 0) and (n < 9)):
                break
        except ValueError:
            print("That's not an integer!")
    return n


main()